pkg/kubelet/cm/dra/plugin/dra_plugin_test.go

```diff
@@ -206,6 +183,62 @@ func TestGRPCConnIsReused(t *testing.T) {
 	// We should have only one entry otherwise it means another gRPC connection has been created
 	require.Len(t, reusedConns, 1, "expected length to be 1 but got %d", len(reusedConns))
 	require.Equal(t, 2, reusedConns[conn], "expected counter to be 2 but got %d", reusedConns[conn])
+
+	t.Run("health_api_reuses_connection", func(t *testing.T) {
+		ctx, cancel := context.WithTimeout(tCtx, 5*time.Second)
+		defer cancel()
+
+		originalConn := plugin.conn
+
+		stream, err := plugin.NodeWatchResources(ctx)
```

Comment:
This `NodeWatchResources` use a logger that is tied to the *parent* test because that is where `tCtx` comes from, not the current sub-tests.